client/asset/eth/eth.go

```diff
@@ -849,7 +823,8 @@ func (w *ETHWallet) Reconfigure(ctx context.Context, cfg *asset.WalletConfig, cu
 	}
 
 	if rpc, is := w.node.(*multiRPCClient); is {
-		if err := rpc.reconfigure(ctx, cfg.Settings); err != nil {
+		walletDir := getWalletDir(w.dir, w.net)
```

Comment:
The `getWalletDir` path seems unnecessary. It will end up being something like `$HOME/.dexc/simnet/assetdb/eth/simnet` and I don't think the last `siment` is necessary. The `keystore` folder already lives there though, is it worth changing now?

Reply:
I've noticed that too.  Not sure if we should change it though.  However, nobody should be using this on mainnet, and we can move our testnet folders around if we do change it.
Maybe someone else has a more decisive opinion.

Reply:
This has always bugged me too. On one hand, the asset package shouldn't make assumptions about what the caller is going to give. On the other hand, as you pointed out, it's stupid with the extra directory.

Anyway, won't change this bork existing wallets?

Reply:
> Anyway, won't this change bork existing wallets?

Yes. Existing testnet eth multiprc wallets.

Reply:
Just leaving for now, can be addressed in another pr if necessary.